pkg/apis/addtoscheme_networkaddonsoperator_v1alpha1.go

```diff
@@ -0,0 +1,10 @@
+package apis
+
+import (
+	cnaov1alpha1 "github.com/kubevirt/cluster-network-addons-operator/pkg/apis/networkaddonsoperator/v1alpha1"
+)
+
+func init() {
+	// Register the types with the Scheme so the components can map objects to GroupVersionKinds and back
+	AddToSchemes = append(AddToSchemes, cnaov1alpha1.SchemeBuilder.AddToScheme)
```

Comment:
This is raceful in case `init` is not sequencially called.

Reply:
well it's used in the same manner in knmstate and KMP, but I didn't find solid evidence that it runs sequentially, so let's be on the safe side.
Done

Reply:
it's done the same in knmstate